CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
+
 # nf-core/methylseq
 
-## v1.6dev - [date]
+## v1.6dev - [2020-02-15]
```

Comment:
We don't know when we will do the release yet :)
```suggestion
## v1.6dev - [date]
```

Reply:
Oh, sorry, I thought it depends on when I open the PR, sorry

Reply:
No problem :)